.buildkite/pipeline.yml

```diff
@@ -28,7 +28,7 @@ steps:
         command: "make lint check-local-changes"
         agents:
           cpu: "6"
-          memory: "6G"
+          memory: "7G"
```

Comment:
This is probably to revert now that you merge `main` and use `agent:ba68d389 (go 1.19.1)`.

Reply:
ty. forgot; removed.